frontend/src/components/ImageWidget.tsx

```diff
@@ -7,10 +7,11 @@ type Props = {
   file: File | undefined;
   summaryBelow: boolean;
   altText: string;
+  scalePct: string;
```

Comment:
recommendation: consider restricting type to allow only possible values

```
scalePct: "auto" | "25%" | "50%" | "75%" | "100%"
```

or use an enum type

```
export enum ScalePct {
   Auto: "auto"
...
}
```

Reply:
I prefer to keep it as is for two reasons:
1. Makes it easier to add values in the future especially if we decide to allow a user to enter any scale value.
2. Makes other parts of the code simpler as we can insert the scale string value immediately as is.